### PR TITLE
[#182994877] Bar graphs with authored data

### PR DESCRIPTION
**Pull request by lublagg**

### CODE REVIEW

File: src/components/graph.tsx [hunk 1]
```diff
@@ -89,10 +91,61 @@ export class Graph extends React.Component<GraphProps, GraphState> {
...
+          return false;
+        }
+      } else {
+        return false;
+      }
+    }
+
+    dataForMultiBar() {
+      const {useAuthoredData, preRecording, usePrediction, prediction} = this.props;
+;     const {data} = this.state;
```
Comment (emcelroy) [inline src/components/graph.tsx:116]: Looks like an errant semicolon here.

File: src/components/graph.tsx [hunk 2]
```diff
@@ -89,10 +91,61 @@ export class Graph extends React.Component<GraphProps, GraphState> {
...
 
+        this.getPlotterType = this.getPlotterType.bind(this);
+        this.dataForMultiBar = this.dataForMultiBar.bind(this);
+        this.useMultiBar = this.useMultiBar.bind(this);
+        this.makeDygraph = this.makeDygraph.bind(this);
         this.dyUpdateProps = ["width", "height", "xMin", "xMax", "yMin", "yMax",
                                 "valuePrecision", "xLabel", "yLabel", "predictionState"];
     }
 
+    useMultiBar() {
```
Comment (emcelroy) [inline src/components/graph.tsx:102]: Could this be simplified to something like the below?

```suggestion
useMultiBar() {
  const { displayType, useAuthoredData, usePrediction } = this.props;
  return displayType === "bar" && (useAuthoredData || usePrediction);
}
```

**Review by emcelroy — CHANGES_REQUESTED** [review]
This looks good and works as expected in the browser.

There's just one tiny blocking change needed: removing that errant semicolon in src/components/graph.tsx.

I also made a suggestion for simplifying the `useMultiBar` function in that file. But feel free to ignore that if you have good reason to stick with your version.

File: src/components/graph.tsx (hunk 1 above)
Comment (lublagg) [reply src/components/graph.tsx]: Good catch!

**Review by emcelroy — APPROVED** [review]
Looks good -- thanks!

**Pull request merged**